components/dropdown/dropdown-menu.js

```diff
@@ -176,6 +176,8 @@ class DropdownMenu extends ThemeMixin(DropdownContentMixin(LitElement)) {
 
 		menu.resize();
 
+		this._closeRadio = false;
```

Comment:
super nit: I'd probably put this with `this._initializingHeight = true;` to have all the property changes grouped